learning_resources/etl/loaders.py

```diff
@@ -335,6 +335,16 @@ def load_course(
             unique_field_value,
         )
         resource_id = deduplicated_course_id or readable_id
+
+        if config.fetch_only:
+            # Do not upsert the course, it should already exist.
+            # Just find it and return it.
+            return LearningResource.objects.filter(
+                readable_id=resource_id,
+                platform=platform,
+                resource_type=LearningResourceType.course.name,
+            ).first()
```

Comment:
Should there be a warning if the course is not found?

Reply:
added